client/devdocs/style.scss

```diff
@@ -199,3 +199,28 @@
 		}
 	}
 }
+
+.docs__post-schedule {
```

Comment:
I think the main devdocs stylesheet is not the right place to introduce component-specific styles.